src/main/java/core/basesyntax/service/impl/operation/OperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.impl.operation;
+
+import core.basesyntax.model.Fruit;
+
+public interface OperationHandler {
+    int MIN_FRUIT_QUANTITY = 0;
```

Comment:
The constant `MIN_FRUIT_QUANTITY` should be encapsulated within the class that uses it. Avoid declaring constants in interfaces unless they are strongly related to the interface's purpose.